hq_superset/services.py

```diff
@@ -144,6 +149,35 @@ def to_sql(df, replace=False):
         raise ex
 
 
+def subscribe_to_hq_datasource(domain, datasource_id):
+    hq_client = HQClient.get_by_domain(domain)
+    if hq_client is None:
+        hq_client = HQClient.create_domain_client(domain)
+
+    hq_request = HQRequest(url=datasource_subscribe(domain, datasource_id))
+    webhook_url = urljoin(
+        request.root_url,
+        url_for('DataSetChangeAPI.post_dataset_change'),
+    )
```

Comment:
Personally I'd put this in another function as well, just so `subscribe_to_hq_datasource` doens't have to know how the `webhook_url` is generated., but I'll leave it up to you...

Reply:
ba6286d